src/compiler/compile/render_dom/wrappers/RawMustacheTag.ts

```diff
@@ -51,9 +53,12 @@ export default class RawMustacheTagWrapper extends Tag {
 
 			const update_anchor = needs_anchor ? html_anchor : this.next ? this.next.var : 'null';
 
-			block.chunks.create.push(b`${html_tag} = new @HtmlTag();`);
+			const parent_element = this.node.find_nearest(/^Element/) as Element;
+			const is_svg = parent_element && parent_element.namespace === namespaces.svg;
+			block.chunks.create.push(b`${html_tag} = new @HtmlTag(${is_svg ? 'true' : 'false'});`);
+
 			if (this.renderer.options.hydratable) {
-				block.chunks.claim.push(b`${html_tag} = @claim_html_tag(${_parent_nodes});`);
+				block.chunks.claim.push(b`${html_tag} = ${is_svg ? '@claim_svg_tag' : '@claim_html_tag'}(${_parent_nodes});`);
```

Comment:
dont see much benefit for differentiating `claim_svg_tag` and `claim_html_tag`, can just do

```js
// for html
html_tag = claim_html_tag(nodes)
// for svg
html_tag = claim_html_tag(nodes, 1)
```

Reply:
OK. I don't have strong opinion here, so I will call `claim_html_tag` as your suggestion.